models/articles.models.js

```diff
@@ -12,4 +12,23 @@ exports.selectArticleByArticleID = (article_id) => {
         }
         return rows[0]
     })
-}
\ No newline at end of file
+}
+
+exports.selectArticles = () => {
+    const commentCount = ` 
+    CAST(
+        (SELECT COUNT(*) FROM comments WHERE comments.article_id = articles.article_id) 
+    AS INTEGER)
+    `
+    const sqlQuery = `
+    SELECT author, title, article_id, topic, created_at, votes, article_img_url, ${commentCount} AS comment_count FROM articles
+    ORDER BY created_at DESC
```

Comment:
Instead of injecting into the SQL - you can put your CAST command straight into the SQL query

Reply:
Ah ok, I thought it might be more readable to split it out but would you say it is best all in one SQL query?